### ISSUE TITLE
ID Localization

### ISSUE THREAD

**Issue by the7thNightmare** [mimoguz/tripeaks-gdx#5]
I want to localize the game to Indonesian Language, how can I help?

**Comment by mimoguz** [comment]
Hi. Do you know about Java i18n resource bundles and property files? If you do, you can find the files [here](https://github.com/mimoguz/tripeaks-gdx/tree/main/android/assets/i8n). If you don't, please take a look at [this post](https://github.com/mimoguz/tripeaks-gdx/issues/4#issuecomment-820993034). Then I'll ask you to translate the landing page text, but first things first 😊

Thank you for your interest.

**Comment by the7thNightmare** [comment]
What's 'chain' mean? Maybe by screenshot you can show it to me where this word appears, thanks.

**Comment by mimoguz** [comment]
Successive removals without using undo or deal button. That line is used on the [end game dialog](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/main/fastlane/metadata/android/en-US/images/phoneScreenshots/4.png).

**Comment by the7thNightmare** [comment]
Like 'streak'?

**Comment by mimoguz** [comment]
Yes

**Comment by the7thNightmare** [comment]
start=Mulai
fromStack=Anda mengambil {0,choice,0#nol kartu|1#satu kartu|1<{0,number,integer} kartu} dari tumpukan.
usedUndo=Anda {0,choice,0#tidak menggunakan batal|1#menggunakan batal satu kali|2#menggunakan batal dua kali|2<menggunakan batal {0,number,integer} kali}.
longestChain=Rentetan terpanjang Anda adalah {0,choice,0#nol kartu|1#satu kartu|1<{0,number,integer} kartu}.
newGame=Mulai permainan baru
newGameShort=Permainan baru
exit=Keluar dari permainan
return=Kembali ke permainan
darkTheme=Gunakan tema gelap
showAll=Perlihatkan semua kartu
won=Anda menang!

**Comment by mimoguz** [comment]
Great! Here is the [test image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/main/i8nTestImages/id_ID_01.png).  

Now a few more lines for the store: 

* A tri peaks solitaire game. 
* An open source tri peaks solitaire game.
* Currently only has randomly generated games.
* Pre-solved games option is on the roadmap.

Also the button names on [this image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/main/fastlane/metadata/android/en-US/images/phoneScreenshots/1.png):
* Deal
* Undo
* Menu

**Comment by the7thNightmare** [comment]
Permainan solitaire tiga puncak.
Permainan solitaire tiga puncak sumber terbuka.
Saat ini hanya ada permainan yang dihasilkan secara acak.
Opsi permainan yang sudah diselesaikan sebelumnya sedang direncanakan.

Ambil
Batal
Menu

**Comment by the7thNightmare** [comment]
For the test image, it's great. Thank you. Also note that in Indonesia 1 to 11 writing's standard is using words, 12 and later is using number, so if you want the words for 3 to 11 is below:
3:Tiga
4:Empat
5:Lima
6:Enam
7:Tujuh
8:Delapan
9:Sembilan
10:Sepuluh
11:Sebelas
Cards:Kartu

**Comment by mimoguz** [comment]
Can you check [the second test](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/main/i8nTestImages/id_ID_02.png)? I added special cases for 0-11.

**Comment by the7thNightmare** [comment]
That's good.

**Comment by mimoguz** [comment]
Can I use just "Keluar" instead of "Keluar dari permainan"?

**Comment by the7thNightmare** [comment]
Yes, you can use it. 'Keluar' is the translation of quit so there's no meaning change about it, the same as return, you can just use 'Kembali' instead the long sentence. Also note for newgame string, you cannot use just 'Mulai' use the long sentence, the same as other string. 'Permainan' is the translation of game.

**Comment by mimoguz** [comment]
Done 👍

Forgot to ask though: I release this on the Play Store too. As is, free, without ads or anything, store page links to this repo. Is it OK for you?

**Comment by the7thNightmare** [comment]
That's okay.

**Comment by mimoguz** [comment]
Hi.

I'm readying a new release with a few new features, so I need new strings. I appreciate your help on this.

## New strings

* No more moves! ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_stalled_dialog.png), displayed when the game is stalled)
* Undo last move ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_stalled_dialog.png))
* Save ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_options_dialog.png))
* Cancel  ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_options_dialog.png)) 
* Start with an empty discard ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_options_dialog.png), [result](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/empty_discard.png))
* New game layout: ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_options_dialog.png))
*Options ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_menu.png)

### Layout options
* Three peaks
* Two peaks and a funnel ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_inverted_2nd_layout.png))
* Diamonds ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_diamonds_layout.png))

### Game description
* An open-source tri peaks solitaire game with multiple layout options.

Thanks!

**Comment by the7thNightmare** [comment]
**New strings**

- Tidak ada langkah lagi!
- Batalkan langkah terakhir
- Simpan
- Batal
- Mulai dengan buangan kosong
- Tata letak permainan baru:
- Pengaturan

**Layout options**

- Tiga Puncak
- Dua Puncak dan Satu Lembah
- Wajik

**Game description**

Permainan solitaire tri-puncak sumber terbuka dengan berbagai opsi tata letak.

> Note: It is best to change 'funnel' to 'valley' 'cause talking about the opposite of peaks is valley, right?

**Comment by mimoguz** [comment]
> Note: It is best to change 'funnel' to 'valley' 'cause talking about the opposite of peaks is valley, right?

🤯 Two peaks and a valley? Peak - valley - peak?

Speaking of which, I think I should change "new game layout" too. Because, is it "**new-game** layout", or "new **game-layout**"? Is that a problem for the translated phrase too?

**Comment by the7thNightmare** [comment]
For Indonesian translation, the two is basically the same, so for me there's nothing wrong with that. But for me, it is best to leave it without hyphen, though it is up to you. (new-game layout is much easier to understand if it for the new game options before the game starts)

**Comment by mimoguz** [comment]
I will not add a hyphen, but change the phrase entirely. Right now it's not clear which one is new, game, or layout. 

English is not my first language, so I may also need a Bad-English to English translator. 🙂

**Comment by mimoguz** [comment]
I'm leaving the string changes to a later release. 

New release is up.

**Comment by mimoguz** [comment]
Hello again.

I want to make some small changes for the next release. Could you have look at these:

* Exit   (Before: "Exit **game**")
* Start with an empty discard **pile**   (Before: "Start with an empty discard")
* <s>Two peaks and a **valley**  (Before: "Two peaks and a **funnel**")</s> As you suggested 🙂. It's already in ID.
* Layout for new games   (Before: "New game layout")
* Your longest chain **had** ...  (Before: "Your longest chain **was** ...")

They may be  already OK on your version though, some (most) of them are because of my rather poor English. 🙂

Thanks.

**Comment by the7thNightmare** [comment]
The translation:

- Keluar
- Mulai dengan tumpukan buangan kosong
- Dua Puncak dan Satu Lembah
- Tata letak untuk permainan baru

There's no change for the last, though.

**Comment by mimoguz** [comment]
👍

**Comment by mimoguz** [comment]
Hi. I want to add one last thing before calling this 1.0. Could you please check the [issue #14](https://github.com/mimoguz/tripeaks-gdx/issues/14)? Thanks.

**Comment by the7thNightmare** [comment]
- Statistik
- Semua
- Permainan:
- Menang:
- Rentetan terpanjang:

**Comment by mimoguz** [comment]
Release created 👍

**Comment by mimoguz** [comment]
[New strings](https://github.com/mimoguz/tripeaks-gdx/issues/27)
